### ISSUE TITLE
storaged 2.5.1 reports wrong LVM2 pool sizes

### ISSUE THREAD

**Issue by mvollmer** [cockpit-project/cockpit#4547]
Because of a bug, it reports the size of a pool as about twice as what it really is. Fixed in storaged master.

Because the pool is reported as too large, Cockpit incorrectly refuses a resize that our tests do.  The pool actually gets larger, but to Cockpit it looks as if it gets smaller, which is not allowed.

**Comment by cockpituous** [comment]
verify/fedora-24
Ooops, it happened again<hr>
```
Traceback (most recent call last):
  File "check-storage-lvm2", line 186, in testLvm
    self.dialog({ "size": 70 })
  File "storagelib.py", line 136, in dialog
    self.dialog_wait_close()
  File "storagelib.py", line 121, in dialog_wait_close
    self.browser.wait_not_present('#dialog')
  File "testlib.py", line 219, in wait_not_present
    return self.wait_js_func('!ph_is_present', selector)
  File "testlib.py", line 210, in wait_js_func
    return self.phantom.wait("%s(%s)" % (func, ','.join(map(jsquote, args))))
  File "testlib.py", line 660, in <lambda>
    return lambda *args: self._invoke(name, *args)
  File "testlib.py", line 683, in _invoke
    raise Error(res['error'])
Error: timeout
```
First occurrence: 2016-06-06T08:48:15.953516 | revision 1d8af97535a28140551e5cf3752d092179fda3ad, [logs](http://fedorapeople.org/groups/cockpit/logs/pull-4545-1d8af975-verify-fedora-24/log.html)
Times recorded: 1
Latest occurrences:

- 2016-06-06T08:48:15.953516 | revision 1d8af97535a28140551e5cf3752d092179fda3ad, [logs](http://fedorapeople.org/groups/cockpit/logs/pull-4545-1d8af975-verify-fedora-24/log.html)

**Comment by stefwalter** [comment]
@mvollmer What is the upstream bug for this known issue?

**Comment by mvollmer** [comment]
This was fixed in storaged 2.5.2 with https://github.com/storaged-project/storaged/pull/58

**Comment by stefwalter** [comment]
This no longer happens on any of our tested operating systems. In addition it hasn't happened in a long time during testing. Closing.
